ddtrace/tracer/tracer.go

```diff
@@ -156,13 +159,23 @@ func newUnstartedTracer(opts ...StartOption) *tracer {
 		rulesSampling:    newRulesSampler(c.samplingRules),
 		prioritySampling: sampler,
 		pid:              strconv.Itoa(os.Getpid()),
+		appTags:          map[string]interface{}{},
 	}
 }
 
 func newTracer(opts ...StartOption) *tracer {
 	t := newUnstartedTracer(opts...)
 	c := t.config
 	t.config.statsd.Incr("datadog.tracer.started", nil, 1)
+	for k, v := range t.globalTags {
+		t.setAppTag(k, v)
```

Comment:
Global tags are not app tags. We are not making any changes to global tags, and no changes to hostname.

Reply:
At this point, we probably won't have a map of app tags, so I think I'll remove this loop, but keep individual app tag settings